fenix/app/src/test/java/org/mozilla/fenix/onboarding/JunoOnboardingFragmentTest.kt

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.onboarding
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.mozilla.fenix.R
+import org.mozilla.fenix.onboarding.view.OnboardingPageUiData
+
+class JunoOnboardingFragmentTest {
+
+    @Test
+    fun `GIVEN first page in the list WHEN sequencePosition called THEN returns the index plus 1`() {
+        val expected = "1"
+        val actual = allKnownPages.sequencePosition(defaultBrowserPageUiData.type)
+
+        assertEquals(expected, actual)
+    }
+
+    @Test
+    fun `GIVEN last page in the list WHEN sequencePosition called THEN returns the index plus 1`() {
+        val expected = "3"
+        val actual = allKnownPages.sequencePosition(notificationPageUiData.type)
+
+        assertEquals(expected, actual)
+    }
+
+    @Test
+    fun `GIVEN all known pages of WHEN sequenceId() called THEN should map to the correct sequence id`() {
+        val expected = "default_sync_notification"
+        val actual = allKnownPages.telemetrySequenceId()
+
+        assertEquals(expected, actual)
+    }
+
+    @Test
+    fun `GIVEN some of the known pages WHEN sequenceId() called THEN should map to the correct sequence id`() {
+        val expected = "default_sync"
+        val actual = allKnownPages.take(2).telemetrySequenceId()
+
+        assertEquals(expected, actual)
+    }
+
+    @Test
+    fun `GIVEN a single known page WHEN sequenceId() called THEN should map to the correct sequence id`() {
+        val expected = "default"
+        val actual = allKnownPages.take(1).telemetrySequenceId()
```

Comment:
Would recommend
```
val actual = listOf(default).telemetrySequenceId()
```
so there is no extra logic in the code path of the test. So we purely test the prod code. 

Same for the test above.